Worksheet_C.md

```diff
@@ -0,0 +1,5 @@
+A) The program is doing a linear search where "i" and "j" duplicate each other.
```

Comment:
It is sort of a linear search, but specifically it is searching for duplicate entries in a list.